packages/material-ui-styles/test/index.spec.tsx

```diff
@@ -99,6 +99,28 @@ function testGetThemeProps(theme: Theme, props: AppBarProps): void {
     // but this would pass anyway
     const alsoClasses = useUnsafeProps(undefined); // runtime: Can't read property color of undefined
   };
+  
+  // default theme
+  interface CustomTheme {
+    attribute: number;
+    [key: string]: any;
+  }
+
+  const validCustomTheme = { attribute: 8, otherStuff: true };
+  const invalidCustomTheme = { otherStuff: true };
+  
+  const style = createStyles((theme: CustomTheme) => ({
+    margin: theme.attribute,
+  }))
+
+  makeStyles<typeof style, CustomTheme>(style, {
```

Comment:
How does this look like without explicitly setting the generic arguments?

Reply:
will add this one too